core/trino-main/src/test/java/io/trino/cost/TestFilterStatsRule.java

```diff
@@ -22,16 +22,13 @@
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.TestInstance;
 
 import static io.trino.spi.type.BigintType.BIGINT;
 import static io.trino.spi.type.DoubleType.DOUBLE;
 import static io.trino.spi.type.IntegerType.INTEGER;
 import static io.trino.sql.ir.Comparison.Operator.EQUAL;
 import static io.trino.testing.TestingSession.testSessionBuilder;
-import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
 
-@TestInstance(PER_CLASS)
```

Comment:
lifecycle might need to remain `StatsCalculatorTester` starts up a queryrunner.
Instead add `Execution(CONCURRENT)`?

Reply:
this test extends BaseStatsCalculatorTest which has `@TestInstance(PER_CLASS)
@Execution(CONCURRENT)`